index.html

```diff
@@ -1,52 +1,3336 @@
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+
+  <title> Portfolio Mobile </title>        
+    </head>
+    <body class="websiteBody">
+      <div class="outer_q1">
+    <div
+      id="q1"
+      data-name="Artboards"
+      class="q1"
+    >
+      <div class="outer_q2">
+      <div
+        id="q2"
+        data-name="Template 5-main-mobile"
+        class="q2"
+      >
+        <div class="outer_q3">
+        <img
+          id="q3"
+          data-name="image_geometry_1"
+          src="images/image_geometry_1.png"
+          alt="image_geometry_1"
+          class="q3"
+        >
+        </div>
+        <div class="outer_q4">
+        <img
+          id="q4"
+          data-name="ic_menu"
+          src="images/ic_menu.png"
+          alt="ic_menu"
+          class="q4"
+        >
+        </div>
+        <div class="outer_q5">
+        <div
+          id="q5"
+          data-name="Group 48"
+          class="q5"
+        >
+          <div class="outer_q6">
+          <div
+            id="q6"
+            data-name="Leo de Andrade"
+            class="q6">
+            <div>Leo de</div>
+<div>Andrade</div>
+          </div>
+          </div>
+          <div class="outer_q7">
+          <div
+            id="q7"
+            data-name="Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me."
+            class="q7"
+          ><div>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.</div>
+<div>If you like what you see and have a project you need coded, don’t hesitate and contact me.</div>
+          </div>
+          </div>
+          <div class="outer_q8">
+          <img
+            id="q8"
+            data-name="Group 19"
+            src="images/Group_19.png"
+            alt="Group 19"
+            class="q8"
+          >
+          </div>
+        </div>
+        </div>
+        <div class="outer_q9">
+        <img
+          id="q9"
+          data-name="image_geometry_2"
+          src="images/image_geometry_2.png"
+          alt="image_geometry_2"
+          class="q9"
+        >
+        </div>
+        <div class="outer_q10">
+        <div
+          id="q10"
+          data-name="Frame 1"
+          class="q10"
+        >
+        </div>
+        </div>
+        <div class="outer_q11">
+        <div
+          id="q11"
+          data-name="Group 119"
+          class="q11"
+        >
+          <div class="outer_q12">
+          <div
+            id="q12"
+            data-name="Projects"
+            class="q12"
+          ><div>Projects</div>
+          </div>
+          </div>
```

Comment:
- [ ] As the previous reviewer pointed out, it is required that this element uses the semantic `section` tag. Kindly implement this and change the `display` property to `grid`. The learning objective is both to be great at responsive design and to meet project-specific requirements 👍🏽